src/apis/auth.ts

```diff
@@ -21,30 +18,22 @@ export const postSignIn = async (reqBody: SignUpReqBody) => {
     data: {
       data: { token },
     },
-  } = await axios.post<SignInBody>(`${serverUrl}/login`, {
+  } = await client.post<SignInBody>(`/login`, {
     email: reqBody.email,
     password: reqBody.password,
   });
```

Comment:
```suggestion
} = await client.post<SignInBody>(`/login`, reqBody);
```
reqBody가 email이랑 password의 키를 가진 객체니깐 그냥 바로 넘겨줘도 되지 않을까요?

Reply:
넵그냥 넘겨도 잘 작동해서 수정해서 커밋했습니다!